docs/Release.md

```diff
@@ -0,0 +1,74 @@
+# Build
+The `share` project uses _Travis CI_. \
+The `.travis.yml` config file can be found in the root of the repository.
+
+
+## Stages and Jobs
+1. **Build**: Java Build with Unit Tests, WhiteSource, Source Clear Scan (SCA)
+2. **Tests**: Executes the E2E tests in `alfresco-tas-share-test`
+3. **Release**: Release by publishing to Nexus.
+4. **Company Release**: Publish docker image for share to Docker Hub.
```

Comment:
Which is the step that pushes the Docker image to Quay?

Reply:
In the build stage, I will add the note.